src/bot.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, screen: Screen, game_stats: GameStats, pick_corpse: bool = Fa
         self._stopping = False
         self._pausing = False
         self._current_threads = []
-        self._no_stash_counter = 0
+        self._no_stash_counter = 5
```

Comment:
Any reason to set this to 5?

Reply:
Just for Debug to trigger stashing when bot start. Set to 0 again